ci/scripts/autobump-dependencies.py

```diff
@@ -344,6 +356,42 @@ def fetch_latest_release(self) -> Release:
             version.parse(latest_version),
         )
 
+    def get_release_notes(self) -> str:
+        current_version = self.current_version
+        latest_version = self.latest_release.version
+
+        if ( current_version == latest_version ):
+            raise Exception(f"""Changelog requested but current and latest versions are the same: {current_version}""")
+
+        releaseNote = f"""
+            [Changelog for {latest_version}](https://www.haproxy.org/download/{HAPROXY_VERSION}/src/CHANGELOG).
+
+            Please also check list of [known open bugs for {latest_version}](https://www.haproxy.org/bugs/bugs-{latest_version}.html).
+        """
```

Comment:
We could also link to the latest "announce" post, which contains some prose on this particular release. It's linked from the haproxy.org homepage and seems to use the search function of the mail archive:
```suggestion
        
        The developer's summary for this release can be found in [the Announcement post for the HAProxy {latest_version} release](https://www.mail-archive.com/search?l=haproxy%40formilux.org&q=announce+subject%3A%22[ANNOUNCE]+haproxy-{latest_version}%22+-subject%3A%22re:%22).
        """
```